_episodes/format.md

```diff
@@ -3,51 +3,260 @@ title: "File format"
+
+A [file format](https://en.wikipedia.org/wiki/File_format) is the structure of a file.
+It determines how the data within the file is organized.
+A File format is also called a file extension.
+Files are usually named as `filename.format`.
+For example, climate.jpeg is a file named climate with [jpeg](https://en.wikipedia.org/wiki/JPEG) format.
+You might know that `jpeg` is a commonly used format for digital images.
+Therefore, the file `climate` is an image.
+
+## Common file format in climate-related domains
+
+In climate-related domains e.g. weather and climate science, Earth observation science, hydrology,
+data can be in many types and for different purposes.
+In this section, we will learn about the common formats of data.
+
+> ## Common file format for geospatial data in your community/research team
+>
+> You work with some data for a project in your community.
+>
+> 1. What is the format of the data?
+> 2. Do you know why the data is in that format?
+>
+{: .discussion}
+
+Here is a list of some common formats:
+
+- [NetCDF](https://www.unidata.ucar.edu/software/netcdf/docs/netcdf_data_model.html):
+  It was originally developed for the Earth science community, but it can be used for **many kinds of data**.
+  NetCDF views the world of scientific data in the same way that a geo-scientist might:
+  ![NetCDF]({{ page.root }}/fig/netcdf-classic-uml.png)
+
+  - There are various quantities such as temperature or elevation
+  located at points at particular coordinates in space and time.
+
+  - The quantities (here temperature or elevation) are stored as netCDF variables.
+  Variables are N-dimensional arrays of **data**, with a name.
+
+  - The coordinates information are stored as netCDF dimensions.
+  Dimensions have a length and a name.
+
+  - A scientist also wants to store supporting information or **metadata**, such as the units, or
+  some information about how the data were produced.
+  The metadata is stored as netCDF attributes that are always single values or one-dimensional arrays.
+
+> ## Why NetCDF format
+>
+> You work with some geospatial data for a project in your community.
+> You choose the NetCDF format for the data.
+> Your colleague asks:
+>
+> 1. Why did you choose the NetCDF format?
+> 2. Does this format help to make the data FAIR?
+>
+>> ## Solution
+>>
+>> 1. You found that NetCDF format is the standard in your research community and
+>> therefore interoperable and reusable for anyone.
+>> Also, the NetCDF format can contain both data and metadata.
+>>
+>> 2. Therefore, yes, the NetCDF format helps to make the data FAIR.
+> {: .solution}
+{: .challenge}
+
+- [GRIB](https://www.wmo.int/pages/prog/www/DPS/FM92-GRIB2-11-2003.pdf):
+  GRIB stands for general regularly-distributed information in binary.
+  It is **commonly used** by the [World Meteorological Organization](https://public.wmo.int/en)(WMO) for weather model data.
+
+  - GRIB coded data consist of a continuous bit-stream made of a sequence of
```

Comment:
This is quite technical. Can we explain that with a figure or so?

Reply:
agree, I removed it, not really needed.